src/components/new-asset-form/new-asset.tsx

```diff
@@ -166,7 +165,7 @@ export const NewAsset = ({
                 }
               >
                 <label className="govuk-label lbh-label" htmlFor="assetId">
-                  Asset ID
+                  Asset ID*
```

Comment:
Added * as it was mentioned to me that it would be helpful to see what the required fields are. Otherwise the only way to know would be to submit the form and see what error messages appear.